.gitignore

```diff
@@ -8,4 +8,6 @@ quaver-venv
 db
 __pycache__
 *.qua
-main.py
\ No newline at end of file
+main.py
+database_lb_test.py
+database_4k_lb_test.py
```

Comment:
This is fine for now but maybe we can just add them to a test folder in the future so that we have examples of how to do things posted in the github